src/main/java/servlets/CreateServlet.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package servlets;
+
+import data.JsonConverter;
+import data.Vendor;
+import data.Account;
+import data.SheetsConverter;
+import static servlets.VendorServlet.updateSheets;
```

Comment:
Do you need a import for that updateSheets method, or can you just import the VenderServlet class itself?

Reply:
I kinda like the static import for this.

Static imports should go above the other imports, though. (So on line 15, with a whitespace above and below)